generator/mavgen_typescript.py

```diff
@@ -81,6 +81,8 @@ def generate_classes(dir, registry, msgs, xml):
                 for field in m.fields:
                     if field.enum:
                         f.write("\tpublic {}!: {};\n".format(field.name, camelcase(field.enum)))
+                    elif field.array_length > 0:
+                        f.write("\tpublic {}!: {}[]; //size: {}\n".format(field.name, ts_types[field.type], field.array_length))
```

Comment:
I think we might have a bug here for strings:

This line will convert 
```<field type="char[249]" name="message">message payload</field>``` 
to 
```public message!: string[]; //size: 249```

We need this to be just string and not array of string